src/translations/sqlb_es_ES.ts

```diff
@@ -2333,7 +2333,7 @@ Do you want to insert it anyway?</source>
     </message>
     <message>
         <location filename="../MainWindow.ui" line="1891"/>
-        <source>Copy the CREATE statement of the item to the clipboard</source>
+        <source>Copy the CREATE statement of the ítem to the clipboard</source>
```

Comment:
@lulol Interesting.  Some of those I was able to find using grep:

### Main window

* [Edit Database Cell](https://github.com/sqlitebrowser/sqlitebrowser/blob/bd0b8637e03c8cbf5628354f151b42cef083254e/src/MainWindow.ui#L951)
* [DB Schema](https://github.com/sqlitebrowser/sqlitebrowser/blob/bd0b8637e03c8cbf5628354f151b42cef083254e/src/MainWindow.ui#L1338)

### Preferences pialog
* [Content](https://github.com/sqlitebrowser/sqlitebrowser/blob/bd0b8637e03c8cbf5628354f151b42cef083254e/src/PreferencesDialog.ui#L362)
* [Symbol limit in cell](https://github.com/sqlitebrowser/sqlitebrowser/blob/bd0b8637e03c8cbf5628354f151b42cef083254e/src/PreferencesDialog.ui#L368)

Does that help? :smile: